<launch>
    <node name="wheels_odometry" pkg="phobos_rover_telemetry" type="wheels_odometry" output="screen">
        <param name="wheel_circumference" value="0.6"/>
        <param name="wheels_spacing" value="0.9"/>
        <param name="impulses_per_rotation" value="1000"/>
    </node>
</launch>
